var/spack/repos/builtin/packages/py-py2cairo/package.py

```diff
@@ -35,7 +35,7 @@ class PyPy2cairo(Package):
 
     extends('python')
 
-    depends_on('cairo+X')
+    depends_on('cairo')
```

Comment:
OK, I can put this back and just have py-pygtk depend on gtkplus+X. The reason I made the change was that py-pygtk would fail with Package cairo requires variant ~X, but spec asked for +X.